src/CMakeLists.txt

```diff
@@ -383,10 +412,10 @@ if(ENABLE_FAST_TOKENIZERS)
     endif()
   endif()
 
-  if(WIN32 AND X86_64)
-    list(APPEND extra_libs "${fast_tokenezers_libs_dir}/icudt70.dll"
-                           "${fast_tokenezers_libs_dir}/icuuc70.dll")
-  endif()
+if(WIN32 AND X86_64)
+  list(APPEND extra_libs "${fast_tokenezers_libs_dir}/icudt70.dll"
+                         "${fast_tokenezers_libs_dir}/icuuc70$<$<CONFIG:Debug>:${CMAKE_DEBUG_POSTFIX}>.dll")
+endif()
```

Comment:
please, return indentation 

why `icudt70.dll` does not have debug suffix?

Reply:
this is data file, not a real dll, icu build doesn't add the postfix for it.